facia/app/utils/FaciaPicker.scala

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import common.GuLogging
+import experiments.{ActiveExperiments, FrontRendering}
+import implicits.Requests._
+import model.PressedPage
+import play.api.mvc.RequestHeader
+
+class FaciaPicker extends GuLogging {
+
+  // we don't yet use these parameters but they will play into dcrCouldRender later on
+  def getTier(faciaPage: PressedPage, path: String)(implicit request: RequestHeader): RenderType = {
+    val participatingInTest = ActiveExperiments.isParticipating(FrontRendering)
+    val dcrCouldRender = false
+
+    val tier = if (request.forceDCROff) {
+      LocalRender
+    } else if (request.forceDCR) {
+      RemoteRender
+    } else if (participatingInTest && dcrCouldRender) {
+      RemoteRender
+    } else {
+      LocalRender
+    }
+
+    logTier(faciaPage, path, participatingInTest, dcrCouldRender, tier)
+
+    tier
+  }
+
+  def logTier(
+      faciaPage: PressedPage,
+      path: String,
+      participatingInTest: Boolean,
+      dcrCouldRender: Boolean,
+      tier: RenderType,
+  )(implicit request: RequestHeader): Unit = {
+    val tierReadable = if (tier == RemoteRender) "dotcomcomponents" else "web";
+    val properties =
+      Map(
+        "participatingInTest" -> participatingInTest.toString,
+        "dcrCouldRender" -> dcrCouldRender.toString,
+        "isFront" -> "true",
+        "tier" -> tierReadable,
```

Comment:
Do we need this? I realise that `msg` is a strangely long and inefficient way to record the place the request was rendered but it follows a convention. Having it also recorded here might tempt people away from the convention.

I don't think there's a _huge_ problem if this does happen though, I'm very much picking nits here.

Reply:
We don't need it no, its more just best practice. Filter queries in elasticsearch tend to be cheaper compared to full text searches.

The old msg format is still there so existing queries would continue to work, no time like the present to improve on conventions though! Especially as this older convention seems to be a bit inconsistent already https://github.com/guardian/frontend/blob/0932505f942c43169a0dec7347447900377c8896/article/app/services/dotcomponents/ArticlePicker.scala#L74